validate_tool.py

```diff
@@ -74,11 +74,11 @@ def main(args):
 
     # validate that the note can be annotated by particular annotator
     example_note = [{
-        "id": "foo",
+        "identifier": "awesome-note",
         "noteType": "loinc:LP29684-5",
-        "patientId": "507f1f77bcf86cd799439011",
-        "text": "On 12/26/2020, Ms. Chloe Price met with Dr. Prescott in Seattle.",
-        "note_name": "testing"
```

Comment:
Ah I add note_name into the notes, because there is an annotationSource field that i need to fill in when storing annotations.

Reply:
Do you use the value of `note_name` to set the value of the property [Annotation.AnnotationSource.ResourceSource.name](https://github.com/nlpsandbox/nlpsandbox-schemas/blob/develop/openapi/commons/components/schemas/ResourceSource.yaml)?

Reply:
I do, I've also thought about just passing in `dataset_id` and `fhir_store_id`

Reply:
Since `note_name` is not part of the note schema, it's confusing to add it here. Wouldn't it be better to generate this value when you insert the annotation into the Data Node, e.g. relying on the value of `Note.identifier`?

Reply:
Ya Ive thought about that as part of the upcoming update I'm going to do.  I just did that in the beginning since I wanted to get an MVP out and it made sense here: https://github.com/nlpsandbox/nlpsandbox-client/blob/develop/nlpsandboxclient/client.py#L57. because Im already passing in `dataset_id` and `fhir_store_id`